app/uk/gov/hmrc/fhregistrationfrontend/views/govuk_wrapper.scala.html

```diff
@@ -52,7 +52,7 @@
   @{
       viewHelpers.headWithTrackingConsent(
         linkElem = Some(linksElement),
-        headScripts = Some(analyticsGA)
+        headScripts = None
```

Comment:
Could you check with @rambabu-posa whether this is right, please?

Reply:
@rambabu-posa please verify

Reply:
@GujjalapudiA Yes its correct. we need to remove that entry. Thanks